src/submission/exceptions/exceptions.ts

```diff
@@ -35,7 +35,7 @@ import { DeepReadonly } from 'deep-freeze';
  * @param programId
  * @returns program and donor level exceptions for this programId
  */
-const queryForExceptions = async (programId: string) => {
+export const queryForExceptions = async (programId: string) => {
```

Comment:
this doesn't need to be exported.

Reply:
Right, forgot to remove from earlier dev work
Updated